test/e2e/cloud/gcp/ha_master.go

```diff
@@ -37,7 +37,7 @@ import (
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
 )
 
-func addMasterReplica(zone string) error {
+func addControlPlaneReplica(zone string) error {
 	framework.Logf(fmt.Sprintf("Adding a new master replica, zone: %s", zone))
```

Comment:
log message still says "master" (There are more of these in the renamed methods)